src/pages/[platform]/build-a-backend/storage/download-files/index.mdx

```diff
@@ -904,10 +907,35 @@ Download tasks are run using `URLSessionTask` instances internally. You can lear
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
+### Specify a bucket
```

Comment:
we can skip this separate section and add it to existing example in first code sample section under download data

Reply:
What's your reasoning for not wanting a separate section for these examples? I want to try to avoid making any individual example overly verbose or complicated. For someone just getting started, they're probably not going to be interested in how to specify a bucket and if we include bucket options there it might create confusion. On the other hand, someone familiar with the library will want to see the multi-bucket options distinctly and in detail.